guides/doc-Installing_Proxy/master.adoc

```diff
@@ -23,7 +23,17 @@ include::common/assembly_performing-additional-configuration-on-capsule-server.a
 // Configuring {SmartProxyServer} with External Services
 include::common/assembly_configuring-external-services.adoc[leveloffset=+1]
 
+include::common/assembly_managing-dhcp-on-smart-proxies.adoc[leveloffset=+1]
+
+include::common/assembly_managing-dns-on-smart-proxies.adoc[leveloffset=+1]
+
 :numbered!:
 
 // {SmartProxyServer} Scalability Considerations
 include::common/modules/ref_capsule-server-scalability-considerations.adoc[leveloffset=+1]
+
+[appendix]
+include::common/modules/ref_dhcp-isc-settings.adoc[leveloffset=+1]
+
+[appendix]
+include::common/modules/ref_dhcp-options-for-network-configuration.adoc[leveloffset=+1]
```

Comment:
I'm guessing this shouldn't be shown for satellite
```suggestion
ifndef::satellite[]
[appendix]
include::common/modules/ref_dhcp-isc-settings.adoc[leveloffset=+1]

[appendix]
include::common/modules/ref_dhcp-options-for-network-configuration.adoc[leveloffset=+1]
endif::[]
```

Reply:
Applied.